### DISCUSSION TITLE
Apple Silicon build?

### DISCUSSION THREAD

**Discussion by rob2go** [mlin/GenomicSQLite#20] [Q&A] [answered]
Any help, please? :-)

2023-02-21 15:08:25 URL:https://raw.githubusercontent.com/mlin/GenomicSQLite/release/test/genomicsqlite_smoke_test.py [31706/31706] -> "-" [1]
-- GenomicSQLite smoke test --
timestamp: Tue Feb 21 14:08:25 2023
platform: macOS-11.7.4-arm64-arm-64bit
uname: posix.uname_result(sysname='Darwin', nodename='RobMBAir.fritz.box', release='20.6.0', version='Darwin Kernel Version 20.6.0: Fri Dec 16 00:34:59 PST 2022; root:xnu-7195.141.49~1/RELEASE_ARM64_T8101', machine='arm64')
python: CPython 3.11.2
cpu: Apple M1
environment:
  PATH=/opt/homebrew/opt/python@3.11/libexec/bin:/Users/roberta/perl5/bin:/opt/homebrew/opt/make/libexec/gnubin:/opt/homebrew/opt/grep/libexec/gnubin:/opt/homebrew/opt/gnu-which/libexec/gnubin:/opt/homebrew/opt/gnu-tar/libexec/gnubin:/opt/homebrew/opt/gnu-sed/libexec/gnubin:/opt/homebrew/opt/gnu-indent/libexec/gnubin:/opt/homebrew/opt/gawk/libexec/gnubin:/opt/homebrew/opt/findutils/libexec/gnubin:/opt/homebrew/opt/coreutils/libexec/gnubin:/opt/homebrew/opt/binutils/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/ncbi/blast/bin:/Library/Apple/usr/bin
sqlite3: v3.40.1
  ATOMIC_INTRINSICS=1 COMPILER=clang-13.0.0 DEFAULT_AUTOVACUUM DEFAULT_CACHE_SIZE=-2000 DEFAULT_FILE_FORMAT=4 DEFAULT_JOURNAL_SIZE_LIMIT=-1 DEFAULT_MMAP_SIZE=0 DEFAULT_PAGE_SIZE=4096 DEFAULT_PCACHE_INITSZ=20 DEFAULT_RECURSIVE_TRIGGERS DEFAULT_SECTOR_SIZE=4096 DEFAULT_SYNCHRONOUS=2 DEFAULT_WAL_AUTOCHECKPOINT=1000 DEFAULT_WAL_SYNCHRONOUS=2 DEFAULT_WORKER_THREADS=0 ENABLE_COLUMN_METADATA ENABLE_FTS3 ENABLE_FTS3_PARENTHESIS ENABLE_FTS4 ENABLE_FTS5 ENABLE_GEOPOLY ENABLE_MATH_FUNCTIONS ENABLE_PREUPDATE_HOOK ENABLE_RTREE ENABLE_SESSION MALLOC_SOFT_LIMIT=1024 MAX_ATTACHED=10 MAX_COLUMN=2000 MAX_COMPOUND_SELECT=500 MAX_DEFAULT_PAGE_SIZE=8192 MAX_EXPR_DEPTH=1000 MAX_FUNCTION_ARG=127 MAX_LENGTH=1000000000 MAX_LIKE_PATTERN_LENGTH=50000 MAX_MMAP_SIZE=0x7fff0000 MAX_PAGE_COUNT=1073741823 MAX_PAGE_SIZE=65536 MAX_SQL_LENGTH=1000000000 MAX_TRIGGER_DEPTH=1000 MAX_VARIABLE_NUMBER=250000 MAX_VDBE_OP=250000000 MAX_WORKER_THREADS=8 MUTEX_PTHREADS SYSTEM_MALLOC TEMP_STORE=1 THREADSAFE=1
genomicsqlite: 
Traceback (most recent call last):
  File "<stdin>", line 526, in <module>
  File "<stdin>", line 72, in main
  File "/opt/homebrew/lib/python3.11/site-packages/genomicsqlite/__init__.py", line 30, in <module>
    _MEMCONN.load_extension(_DLL)
sqlite3.OperationalError: dlopen(libgenomicsqlite.dylib, 10): image not found

**Comment by mlin** [comment, marked as answer]
No Apple Silicon support yet, sorry. I don't have a suitable machine to develop & test on.
